release.sh

```diff
@@ -71,7 +71,14 @@ then
   exit 1
 fi
 
-echo Copying over base Dockerfiles
+printf "\n\nDeleting old container images\n\n"
+
+until docker images | grep "containers.schibsted.io/finntech/node" | awk "{print $3}" | sort | uniq | xargs docker rmi -f
```

Comment:
also, the awk program must be wrapped in single (not double) quotes to not print everything (at least on my mac and an ubuntu server I tested with) for whatever reason